src/controllers/newAccountController.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from '../models/customerModel'
+import { createAccount } from "../models/accountModel";
+import { Account } from "../models/accountModel"
+import { addMoney } from "../models/accountModel";
+import generateAccountNumber from "../utilities/generateAccountNumber";
+
+const buildAccountObject = (user: User): Account => {
+    const accountNumber = generateAccountNumber
```

Comment:
Update the export from `generateAccountNumber` to not call the function, then call it here instead